### ISSUE TITLE
Style/MixinUsage do not detect all the offenses

### ISSUE THREAD

**Issue by Ana06** [bbatsov/rubocop#5171]
`Style/MixinUsage` cop doesn't complain about all the offenses.

Foe example, with the following code in the file `request_controller.rb`:

``` Ruby
require_dependency 'event/all'

include MaintenanceHelper

class RequestController < ApplicationController
  # There is some code here...
end
```

Rubocop doesn't complains.

However, if you remove the first line, so:

``` Ruby
include MaintenanceHelper

class RequestController < ApplicationController
  # There is some code here...
end
```

It does complain:

```
request_controller.rb:1:1: C: Style/MixinUsage: include is used at the top level. Use inside class or module.
include MaintenanceHelper
^^^^^^^^^^^^^^^^^^^^^^^^^
```
--------

## Expected behavior

Both cases should fail, as in the first case the `include` is also not inside a class or a module.

## Actual behavior

Only the second case fails.

## RuboCop version

```
$ rubocop -V
0.51.0 (using Parser 2.4.0.0, running on ruby 2.4.2 x86_64-linux-gnu)
```

**Comment by koic** [comment]
Thanks for opening issue.
This issue related to #4885 and fixed by #4941. Thank you.

**Comment by Ana06** [comment]
@koic 

> Thanks for opening issue.
This issue related to #4885 and fixed by #4941. Thank you.

True, thanks for the fix! 😉
